### ISSUE TITLE
Twisted.Web in 60 seconds / WSGI contains a typo

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5438]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#5438|
|Type|enhancement|
|Created|2011-12-21 19:34:53Z|

it says

```
Note thate <code>WSGIResource</code> is a multithreaded WSGI container.
```

but I suppose it's meant to say

```
Note that a <code>WSGIResource</code> is a multithreaded WSGI container.
```

Patch attached.

Attachments:

* [web-in-60-wsgi.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/849/8499fc5ad3be9269c190fb44d834ead8e0a436cc/0c82c532226504f50ebb9474a2049b208793b28a.diff) (655 bytes) - added by hynek on 2011-12-21 19:35:15Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5438 5438
type__enhancement enhancement
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1324496093000000 1324496093000000
changetime__1324560329000000 1324560329000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!  I'm going to leave the "a" out because I think it reads better that way.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [33399]) Fix a typo in the WSGI howto

Author: hynek
Reviewer: exarkun
Fixes: [#5438](https://github.com/twisted/twisted/issues/5438)
